RELEASE-NOTES.md

```diff
@@ -1,5 +1,33 @@
 # Prefect Release Notes
 
+## Release 2.10.17
+
+### Improved Prefect Tutorial
+
+Unleash the full potential of Prefect with the [newly improved tutorial](https://docs.prefect.io/2.10.17/tutorial/). Elevate your learning experience and embark on a transformative journey that equips you with the skills to orchestrate and observe all of your workflows.
+
+### Enhancements
+- Update Prefect client to follow redirects by default — https://github.com/PrefectHQ/prefect/pull/9988
+- Always show checkboxes rather than animating them on hover — https://github.com/PrefectHQ/prefect-ui-library/pull/1490
```

Comment:
```suggestion
- Always show checkboxes on list items, rather than animating them on hover — https://github.com/PrefectHQ/prefect-ui-library/pull/1490
```